styles/normalize.css

```diff
@@ -347,3 +347,23 @@ template {
 [hidden] {
     display: none;
 }
+
+.slick-slide img {
+  width: 100%;
+}
+
+.slick-prev {
+  left: 0;
+  z-index: 1000;
```

Comment:
not a big deal, but I am not a fan of adding really big z-index off the bat since from my experience in time in certain situations the number got so big for no reason that I can understand; unless this is needed to go over a library index that is almost this high